views/HakaiWatershedsStreamStations.sql

```diff
@@ -0,0 +1,342 @@
+CREATE
+OR REPLACE VIEW erddap."HakaiWatershedsStreamStations" AS
+SELECT
+    *
```

Comment:
These `select * from` lines can come out as they don't add anything

Reply:
Hum I can just use 
``` sql
CREATE
OR REPLACE VIEW erddap."HakaiWatershedsStreamStations" AS
FROM 
...
```
Never tried that haha

Reply:
oh I see what I you